### PR TITLE
1183 reset settings

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

**Comment by pchelle** [comment]
@Yuri05 
appveyor failed because TLF was not updated yet (`resetTLFSettingsToDefault()` was not available).
Can we re-run appveyor ?

**Comment by Yuri05** [comment]
> @Yuri05 appveyor failed because TLF was not updated yet (`resetTLFSettingsToDefault()` was not available). Can we re-run appveyor ?

Done, succeedeed on rebuild :)

File: R/reportingengine-env.R [hunk 1]
```diff
@@ -352,97 +228,21 @@ getStatisticsFromType <- function(statisticsType) {
...
-  validateIsInteger(n)
-  reEnv$defaultMCRepetitions <- n
-  return(invisible())
-}
+# Save a sync version of re-env as an .RData file
+newEnv <- reEnv
+save(
+  "newEnv",
+  file = file.path(system.file(package = "ospsuite.reportingengine"), "extdata", "re-env.RData")
+)
```
Comment (Yuri05) [inline R/reportingengine-env.R:247]: When is this called?
We should assume that the package installation location is read-only and don't write anything there during runtime.

Comment (pchelle) [reply R/reportingengine-env.R:247]: It is called when I update the documentation of the package: `devtools::document()` and not exported/exposed to the user.

**Review by Yuri05 — APPROVED** [review]

**Comment by Yuri05** [comment]
@pchelle with this PR and the latest tlf I can retry to generate the Quali report, correct?

**Pull request merged**